app/controllers/api/articles_controller.rb

```diff
@@ -10,4 +10,20 @@ def show
   rescue ActiveRecord::RecordNotFound => e
     render json: { message: 'Something went wrong, this article was not found' }, status: 404
   end
+
+  def create
+    article = Article.create(article_params)
+    if article.persisted?
+      render json: { message: 'Your article was successfully created' }, status: 201
+    else
+      render json: { message: 'Something went wrong!' }, status: 422
```

Comment:
use the active record errors that you get if a title isn't added and so on. Put a debugger and get your way through to the specific errors.